src/elements/content-sidebar/activity-feed/task-new/TaskStatus.js

```diff
@@ -29,30 +28,43 @@ const StatusIcon = ({ status, ...rest }: { status: TaskStatus }) => {
             return <IconReject width={ICON_SIZE} height={ICON_SIZE} {...rest} />;
         case TASK_NEW_NOT_STARTED:
         case TASK_NEW_IN_PROGRESS:
-            return <IconPending width={ICON_SIZE} height={ICON_SIZE} {...rest} />;
         default:
             return null;
     }
 };
 
 const StatusMessage = ({ status }: { status: TaskStatus }) => {
+    let statusMessage;
     switch (status) {
         case TASK_NEW_APPROVED:
```

Comment:
Seems like this could be simplified to a map with key/value of status/message and moved to a constant.